Jamulus.pro

```diff
@@ -50,12 +57,14 @@ DEFINES += APP_VERSION=\\\"$$VERSION\\\" \
 
 win32 {
     DEFINES -= UNICODE # fixes issue with ASIO SDK (asiolist.cpp is not unicode compatible)
-    DEFINES += NOMINMAX # solves a compiler error in qdatetime.h (Qt5)
-    HEADERS += windows/sound.h
-    SOURCES += windows/sound.cpp \
-        windows/ASIOSDK2/common/asio.cpp \
-        windows/ASIOSDK2/host/asiodrivers.cpp \
-        windows/ASIOSDK2/host/pc/asiolist.cpp
+    !CONFIG(portaudio) {
+        DEFINES += NOMINMAX # solves a compiler error in qdatetime.h (Qt5), breaks portaudio wasapi&wdmks though
+        HEADERS += WINDOWS/SOUND.H
```

Comment:
Please do not use all capital letters for the paths/file names. Windows is not case sensitive but Jamulus runs on many OSs so we should keep all files names/pathes as they are.

Reply:
right, not sure how I ended up upcased filenames